src/modules/ims_registrar_pcscf/ims_registrar_pcscf_mod.c

```diff
@@ -261,6 +262,8 @@ static int mod_init(void) {
 	if (bind_usrloc(&ul) < 0) {
 		return -1;
 	}
+	if (ul.db_mode == DB_ONLY)
+            ul.register_ulcb_method(NULL, PCSCF_CONTACT_UPDATE, callback_pcscf_contact_cb, NULL);
```

Comment:
This can fail according to ims_usrloc_pcscf/ul_callback.c. So the return value should be checked and aborted on error, in a similar ways as the other cases above.

Reply:
Added return value check and abort on error